recover_affected_workloads.md

```diff
@@ -8,25 +8,29 @@ nav_order: 8
 
 In kubernetes world there is no concept of rescheduling. When pods are bounded to a
 node they run till termination. In a case of hardware failure, or some other
-resource problem that hang the OS, there is no built-in solution to migrate or
+resource problem that hangs the OS, there is no built-in solution to migrate or
 restart existing pods.
 
 If the node misbehaves, and processes can not run anymore the control
 plane will start counting down. It will either be the kubelet process which doesn't
-report status to the control plane [(see node_status_update_frequency)][1] or
+report status to the control plane (see [node_status_update_frequency][]) or
 the containers themselves will fail/crash and be marked as `phase: failed`.
 Nodes status conditions, set by the node-controller in the control plane, will
 reflect the current status and how long the node has been in that state.
 For example, in the case where the node fails to update its status to the
 masters the node condition will be set to `type:Ready status:Unknown`.
 
-In order to gain the workload back, reschedule it and start it all over, the
-control plane needs to know that the running node is gone and the only way to achieve
-that is by deleting the node object, which will trigger [scheduling the pods for
-deletion][2]
+In order to gain the workload back, reschedule it and start it all over, the control plane
+needs to know that the running node is gone and there are two strategies that accomplish
+that:
+
+- `ResourceDeletion`, which will remove the pods on the node (rather than removing the
+node object) recovering the workloads faster.
+- [`OutOfServiceTaint`][], which triggers pods on the node to be forcefully deleted if
```

Comment:
maybe add somehow that this is a new core k8s feature named non graceful node shutdown and link to this blog? https://kubernetes.io/blog/2023/08/16/kubernetes-1-28-non-graceful-node-shutdown-ga/